### PR TITLE
#Sql tasks. Alexander Grebennikov

### PR DESCRIPTION
**Pull request by mollyfox6996**
https://travis-ci.org/github/mollyfox6996/db-assignments/builds/710480356

### CODE REVIEW

File: task/sql-tasks.js [hunk 1]
```diff
@@ -130,7 +198,14 @@ async function task_1_8(db) {
  *
  */
 async function task_1_9(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            CustomerID as "CustomerID",
+            ContactName as "ContactName"
+        FROM Customers
+        WHERE ContactName LIKE 'F%' AND ContactName LIKE '___n%'
```
Comment (IlyaMokin) [inline task/sql-tasks.js]: you can make it with only one 'LIKE' operator

File: task/sql-tasks.js [hunk 2]
```diff
@@ -281,7 +476,30 @@ async function task_1_21(db) {
...
+            ON Products.ProductID = OrderDetails.ProductID
+        WHERE OrderDetails.UnitPrice = (
+            SELECT
+                MAX(OrderDetails.UnitPrice) 
+            FROM Customers c
+            INNER JOIN Orders
+                ON Orders.CustomerID = c.CustomerID
+            INNER JOIN OrderDetails
+                ON Orders.OrderID = OrderDetails.OrderID
+            WHERE c.CompanyName = Customers.CompanyName)
```
Comment (AlexMokin) [inline task/sql-tasks.js]: company name doesn't uniquely identify a customer. Also this field doesn't have an index so the query is slow

**Review by AlexMokin — COMMENTED** [review]
Looks good. See comments.
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by AlexMokin** [comment]
looks good. Accepted!

**Pull request closed without merging**